### DISCUSSION TITLE
checkedKeys 和 selectedKeys 是什么关系？

### DISCUSSION THREAD

**Discussion by Mr-LiuDC** [vbenjs/vue-vben-admin#2720] [Q&A] [answered]
作为一个前端新手，请教下 BasicTree 里面的两个属性 checkedKeys 和 selectedKeys 是什么关系，怎么使用？文档说一个是 `勾选的节点`，一个是 `选中的节点`，感觉是一个意思呢。

**Comment by ghost** [comment, marked as answer]
selectedKeys 表示选中的文字
<img width="420" alt="image" src="https://user-images.githubusercontent.com/106857035/235412959-00bf7941-15d9-4fa9-a697-10be9a9760c7.png">
